### PR TITLE
Disable testing upgrade endpoint

### PR DESCRIPTION
**Pull request by Zillode**

### CODE REVIEW

File: test/http_test.go [hunk 1]
```diff
@@ -159,7 +159,8 @@ func TestGetJSON(t *testing.T) {
 		}
 
 		if ct := res.Header.Get("Content-Type"); ct != "application/json; charset=utf-8" {
-			t.Errorf("Incorrect Content-Type %q for %q", ct, path)
+			// Could fail if disconnected ("couldn't fetch release information")
+			t.Errorf("Incorrect Content-Type %q for %q: %s", ct, path)
```
Comment (calmh) [inline test/http_test.go]: Spurious %s?

**Comment by calmh** [comment]
Ah... In fact, I think we should remove that URL from the list, we shouldn't depend on Github being reachable for the tests to pass.

**Comment by Zillode** [comment]
Dunno, testing whether we can upgrade seems useful:)

**Comment by calmh** [comment]
But now the commit message is misleading :sunglasses:

**Pull request merged**